pkgs/development/haskell-modules/configuration-ghc-8.4.x.nix

```diff
@@ -41,4 +41,1041 @@ self: super: {
+    version         = "2.4.1";
+    sha256          = "1kzrl9njvkbvxylk9jg61vy3ksmxmzymci5hdp0ilpsah4620yjx";
+  });
+
+  tasty = overrideCabal super.tasty (drv: {
+    ## On Hackage, awaiting for import
+    ## 
+    ##     • Could not deduce (Semigroup (Traversal f))
+    ##         arising from the superclasses of an instance declaration
+    ##       from the context: Applicative f
+    version         = "1.0.0.1";
+    sha256          = "0ggqffw9kbb6nlq1pplk131qzxndqqzqyf4s2p7576nljx11a7qf";
+  });
+
+  test-framework = overrideCabal super.test-framework (drv: {
+    ## On Hackage, awaiting for import
+    ## 
+    ## Setup: Encountered missing dependencies:
+    ## time >=1.1.2 && <1.6
+    version         = "0.8.2.0";
+    sha256          = "1hhacrzam6b8f10hyldmjw8pb7frdxh04rfg3farxcxwbnhwgbpm";
+    ## Setup: Encountered missing dependencies:
+    ## HUnit >=1.2,
+    ## QuickCheck >=2.3 && <2.10,
+    doCheck         = false;
+    ## curl: (22) The requested URL returned error: 404 Not Found
+    ## error: cannot download test-framework-0.8.2.0-r3.cabal from any mirror
+    editedCabalFile = null;
+  });
+
+  xml-conduit = overrideCabal super.xml-conduit (drv: {
+    ## On Hackage, awaiting for import
+    ## 
+    ##     • No instance for (Semigroup Attributes)
+    ##         arising from the superclasses of an instance declaration
+    ##     • In the instance declaration for ‘Monoid Attributes’
+    version         = "1.7.1.0";
+    sha256          = "1c4ip76qgqjdyf77h97mf3yxdimv7m5ma5v20wchn9qjmbkr8ffa";
+  });
+
+  yaml = overrideCabal super.yaml (drv: {
+    ## On Hackage, awaiting for import
+    ## 
+    ##     • No instance for (Semigroup (YamlParser a))
+    ##         arising from the superclasses of an instance declaration
+    ##     • In the instance declaration for ‘Monoid (YamlParser a)’
+    version         = "0.8.28";
+    sha256          = "0swgkzkfrwj0ac7lssn8rnrdfmh3lcsdn5fbq2iwv55di6jbc0pp";
+  });
+
+
+  ## Upstreamed
+
+  constraints = overrideCabal super.constraints (drv: {
+    ## Upstreamed, awaiting a Hackage release
+    ## 
+    ##     • Could not deduce (Semigroup (Dict a))
+    ##         arising from the superclasses of an instance declaration
+    ##       from the context: a
+    src = pkgs.fetchgit {
+      url    = "https://github.com/ekmett/constraints";
+      rev    = "07e8ace4d4a842cf450b346ccc75829bd310f4a5";
+      sha256 = "11l1hvnvnmzd3zxwddz5di4h5iql47gbwjz73x2kdy21avr85i4v";
+    };
+    ## Setup: Encountered missing dependencies:
+    ## hspec >=2, semigroups >=0.11 && <0.19
+    libraryHaskellDepends = drv.libraryHaskellDepends ++ [ self.semigroups self.hspec ];
+  });
+
+  free = overrideCabal super.free (drv: {
+    ## Upstreamed, awaiting a Hackage release
+    ## 
+    ##     • Could not deduce (Semigroup (IterT m a))
+    ##         arising from the superclasses of an instance declaration
+    ##       from the context: (Monad m, Monoid a)
+    src = pkgs.fetchgit {
```

Comment:
Would it be possible to use `fetchGitHub` instead of `fetchgit`? It's a minor optimization, granted, but `fetchGitHub` downloads a single tarball rather than fetching each file individually and is therefore more efficient.

Reply:
Sure, I'll do that.

Reply:
Done.